transpose.json

```diff
@@ -0,0 +1,186 @@
+{
+  "#": [
+      "JSON doesn't allow for multi-line strings, so all multi-line input is ",
+      "presented here as arrays of strings. It's up to the test generator to join the ", 
+      "lines together with line breaks."
+  ],
```

Comment:
That seems like a good wording. I copied the text from the existing JSON data from [food-chain](https://github.com/exercism/x-common/blob/master/food-chain.json) by the way.